src/xbot/common/math/PIDPropertyManager.java

```diff
@@ -129,4 +146,12 @@ public boolean getEnableDerivativeThreshold() {
     public void setEnableDerivativeThreshold(boolean isEnabled) {
         propEnableDerivativeThreshold.set(isEnabled);
     }
+    
+    public boolean getEnableTimeThreshold() {
+        return propEnableTimeThreshold.get();
```

Comment:
Honestly, I'm not sure. I initially added the enable/disable for the other tolerances thinking we might, but we haven't so far.

On the other hand, we haven't done a lot of PID tuning yet, so I'm thinking we should keep these around for a little bit longer to see if we use them at all.